tap_s3_csv/conversion.py

```diff
@@ -111,6 +111,9 @@ def generate_schema(samples):
     to_return = {}
     counts = count_samples(samples)
 
+    if not counts:
+        return to_return
```

Comment:
okay so if generate schema returns here, won't the code relying on the schema then fail? Have you tested it?

Reply:
My question is still not addressed

Reply:
the case we're trying to handle is when counts is None. The unit test I added handles that case.

Reply:
**won't the code relying on the schema then fail?**